custom-instrumentation/attributes-and-events/README.md

```diff
@@ -0,0 +1,45 @@
+# Sample Attributes & Events Application
+
+This example provides an application to demonstrate how to share custom attributes and events. This application is helpful for users who want to send custom attributes and events in a web framework transaction.
+
+## Getting Started
+
+1. Clone or fork this repository.
+2. Navigate to this example's sub directory
+   ```
+   cd newrelic-node-examples/custom-instrumentation/attributes-and-events
+   ```
+3. Install dependencies and run application.
+   ```
+   npm install
+   cp env.sample .env
+   # Fill out `NEW_RELIC_LICENSE_KEY` in .env and save 
+   # Start the application
+   npm start
+   ```
+4. Make requests to the application. Route names include: 'custom-attribute', 'custom-attributes', 'custom-span-attribute', 'custom-span-attributes', and 'custom-event'.
+   ```
+   curl -X POST http://localhost:3000/custom-attribute
+   ```
+
+## Exploring Telemetry
+
+After sending a few requests, navigate to your application in APM & Services. Locate the 'Example Attributes & Events App' service and then select Transactions on the left-side of the screen. You should see your requests in a few minutes in a similar fashion to below.
+
+![1721149128373](image/README/1721149128373.png)
```

Comment:
I _think_ you have to add `./` before image. I know this works in IDE but if you try to render this in this PR it's not working